app-antivirus/clamtk/clamtk-5.25.ebuild

```diff
@@ -0,0 +1,170 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PLOCALES="af ar ast az bg bs ca cs da de el_GR en_AU en_CA en_GB eo es eu fi fo fr ga gl he hr hu id it ja km ko lo lt lv mr ms nb nl nl_BE nn pa pl pt pt_BR ro ru si sk sl sr@latin sv ta te th tr ug uk uz zh_CN zh_TW"
+PLOCALE_BACKUP="en_GB"
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit desktop l10n perl-functions python-single-r1 xdg-utils
+
+MY_PV_CINNAMON="0.04"
+MY_PV_GNOME="0.02"
+MY_PV_KDE="0.18"
+MY_PV_XFCE="0.06"
+
+DESCRIPTION="A graphical front-end for ClamAV"
+HOMEPAGE="https://dave-theunsub.github.io/clamtk/"
+SRC_URI="https://bitbucket.org/davem_/clamtk/downloads/${P}.tar.xz
+	cinnamon? ( https://bitbucket.org/davem_/nemo-sendto-${PN}/downloads/nemo-sendto-${PN}-${MY_PV_CINNAMON}.tar.xz )
+	gnome? ( https://bitbucket.org/davem_/${PN}-gnome/downloads/${PN}-gnome-${MY_PV_GNOME}.tar.xz )
+	kde? ( https://bitbucket.org/davem_/${PN}-kde/downloads/${PN}-kde-${MY_PV_KDE}.tar.xz )
+	xfce? ( https://bitbucket.org/davem_/thunar-sendto-${PN}/downloads/thunar-sendto-${PN}-${MY_PV_XFCE}.tar.xz )"
+
+LICENSE="|| ( Artistic GPL-1+ )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="cinnamon gnome kde +nls xfce"
+REQUIRED_USE="gnome? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="app-antivirus/clamav
+	dev-perl/File-chdir
+	dev-perl/Gtk2
+	dev-perl/JSON
+	dev-perl/LWP-Protocol-https
+	dev-perl/LWP-UserAgent-Cached
+	dev-perl/Locale-gettext
+	dev-perl/Text-CSV
+	dev-perl/glib-perl
+	dev-perl/libwww-perl
+	virtual/perl-Digest-MD5
+	virtual/perl-Digest-SHA
+	virtual/perl-Encode
+	virtual/perl-MIME-Base64
+	virtual/perl-Time-Piece
+	cinnamon? ( gnome-extra/nemo )
+	gnome? ( dev-python/nautilus-python[${PYTHON_USEDEP}]
+		gnome-base/nautilus )
+	kde? ( kde-apps/dolphin:5 )
+	xfce? ( xfce-base/thunar )"
+
+DEPEND="nls? ( sys-devel/gettext )"
+
+DOCS=( CHANGES credits.md DISCLAIMER README.md )
+
+pkg_setup() {
+	if use gnome; then
+		python-single-r1_pkg_setup
+	fi
+}
+
+src_prepare() {
+	src_prepare_locales() {
```

Comment:
please rename it to `remove_locale` since you changed logic.
get rid of src_ prefix, it's a bit confusing.
or follow mgorny's advice and get rid of locale stuff completely and install everything =)

Reply:
I don't think, that src_ is in any way confusing, but changed, as requested.